### ISSUE TITLE
Fail to train monolingual model for Chinese

### ISSUE THREAD

**Issue by NLP-hua** [HLTCHKUST/BiToD#5]
Hello, I am trying to train the monolingual model using the given run.sh script.
I have successfully trained the monolingual model for English using mT5. The evaluation results are:

TSR: 66.98      DSR:   43.44       API_acc:   65.09        BLEU:  38.40          JGA : 68.54

This result is comparable to the result reported in the paper and seems to be ok.

However, I failed to train the monolingual model for Chinese. During the training, I got the following information:

{' Loss ': nan, 'Learning_rate ': 0.0004959291187739464, 'epoch': 0.38}
{' Loss ': nan, 'Learning_rate ': 0.000483955938697318, 'epoch': 0.57}
{' Loss ': nan, 'Learning_rate ': 0.00047198275862068967, 'epoch': 0.77}
{' Loss ': nan, 'Learning_rate ': 0.00046000957854406134, 'epoch': 0.96}
{' Loss ': nan, 'Learning_rate ': 0.00044803639846743295, 'epoch': 1.15}

The loss was always Nan...

I don't know why. Is there anything special about training Chinese monolingual models?
Looking forward to your reply :)

**Comment by zlinao** [comment]
hmm, do you follow the exact same hyperparameter?  If so, try to turn off fp16
